railties/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Deprecate `SOURCE_ANNOTATION_DIRECTORIES` environment variable used by `rails notes`
+    in favor of using `config.annotations.register_directories`
+
+    Deprecate `rake notes` in favor of `rails notes`
+
+    Deprecate `rails notes:custom`, `rails notes:optimize`, `rails notes:todo`,
+    and `rails notes:fixme` in favor of `rails notes -a`
```

Comment:
So you'd do `rails notes -a fixme` or `rails notes -a FIXME`?

Reply:
`rails notes -a FIXME` since it's case sensitive